build.gradle

```diff
@@ -0,0 +1,354 @@
+        html.enabled = true
+    }
+}
+
+jacoco {
+    toolVersion = "0.7.5.201505241946"
+}
+
+dependencies {
+    compile 'com.google.guava:guava:15.0'
+    compile 'com.github.samtools:htsjdk:2.4.1'
+    //tools dependency for doclet
+    compile(files(((URLClassLoader) ToolProvider.getSystemToolClassLoader()).getURLs()))
+    testCompile 'org.testng:testng:6.9.10'
+}
+
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
+
+final isRelease = Boolean.getBoolean("release")
+final gitVersion = gitVersion().replaceAll(".dirty", "")
+version = isRelease ? gitVersion : gitVersion + "-SNAPSHOT"
+
+logger.info("build for version:" + version)
+group = 'com.github.broadinstitute'
+
+defaultTasks 'all'
+
+task all(dependsOn: ['jar', 'libJar', 'documentAll', 'documentStandardOptions', 'picardZip'])
+
+jar {
+    manifest {
+        attributes 'Implementation-Title': 'Picard',
+                'Implementation-Vendor': 'Broad Institute',
+                'Implementation-Version': version
+    }
+}
+
+import org.gradle.internal.os.OperatingSystem;
+
+tasks.withType(Test) {
+    outputs.upToDateWhen { false } // tests will always rerun
+    useTestNG()
+
+    // set heap size for the test JVM(s)
+    minHeapSize = "1G"
+    maxHeapSize = "2G"
+
+    jvmArgs '-Djava.awt.headless=true'  //this prevents awt from displaying a java icon while the tests are running
+
+    if (System.env.CI == "true") {  //if running under a CI output less into the logs
+        int count = 0
+
+        beforeTest { descriptor ->
+            count++
+            if (count % 100 == 0) {
+                logger.lifecycle("Finished " + Integer.toString(count++) + " tests")
+            }
+        }
+    } else {
+        // show standard out and standard error of the test JVM(s) on the console
+        testLogging.showStandardStreams = true
+        beforeTest { descriptor ->
+            logger.lifecycle("Running Test: " + descriptor)
+        }
+
+        // listen to standard out and standard error of the test JVM(s)
+        onOutput { descriptor, event ->
+            logger.lifecycle("Test: " + descriptor + " produced standard out/err: " + event.message)
+        }
+    }
+
+    testLogging {
+        testLogging {
+            events "skipped", "failed"
+            exceptionFormat = "full"
+        }
+        afterSuite { desc, result ->
+            if (!desc.parent) { // will match the outermost suite
+                println "Results: ${result.resultType} (${result.testCount} tests, ${result.successfulTestCount} successes, ${result.failedTestCount} failures, ${result.skippedTestCount} skipped)"
+            }
+        }
+    }
+}
+// This is a hack to disable the java 8 default javadoc lint until we fix the html formatting
+if (JavaVersion.current().isJava8Compatible()) {
+    tasks.withType(Javadoc) {
+        options.addStringOption('Xdoclint:none', '-quiet')
+    }
+}
+
+test {
+    description = "Runs the unit tests other than the SRA tests"
```

Comment:
We had separate `test` and `tasks.withType(Test)` configurations in htjsdk because we had multiple tasks of type test, but since there's only one here you could just roll the above configuration into this task.